### PR TITLE
Add first Dutch translation

### PR DESCRIPTION
**Pull request by studio1902**
This is the first Dutch translation. I translated it without using v3 IRL but it should be a pretty solid basis. Will do additional PR's with bug fixes when I start using V3 more in the coming weeks.

### CODE REVIEW

File: resources/lang/nl/fieldtypes.php [hunk 1]
```diff
@@ -0,0 +1,140 @@
+<?php
+
+return [
+    'array' => [
+        'config' => [
+            'keys' => 'Stel de array keys (variabelen) en optionele labels in.',
+            'mode' => 'Dynamische modus geeft de gebruike rcontroler over de data terwijl keyed modus dat niet doet.',
```
Comment (plompd) [inline resources/lang/nl/fieldtypes.php:7]: TYPO: gebruike rcontroler => gebruikte controler

File: resources/lang/nl/fieldtypes.php [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+            'mode' => 'Kies je voorkeur voor layoutstijl.',
+            'restrict' => 'Voorkom dat gebruikers naar andere mapen kunnen bladeren.',
+        ],
+    ],
+    'bard' => [
+        'config' => [
+            'allow_source' => 'Schakel de optie in dat gebruikers de HTML-broncode kunnen bekijken tijdens het schrijven.',
+            'buttons' => 'Kies welke knoppen er in de toolbar zitten.',
+            'container' => 'Kies welke asset-container er voor dit veld gebruikt moet worden.',
+            'fullscreen' => 'Schakel de optie in dat mensen voleldige schermweergave kunnen gebruiken.',
```
Comment (plompd) [inline resources/lang/nl/fieldtypes.php:25]: TYPO: voleldige => volledige

File: resources/lang/nl/fieldtypes.php [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+            'link_noopener' => 'Gebruik `rel="noopener"` op alle links.',
+            'link_noreferrer' => 'Gebruik `rel="norefferer"` op alle links.',
```
Comment (plompd) [inline resources/lang/nl/fieldtypes.php:27]: TYPO: norefferer => noreferrer

Comment (studio1902) [reply resources/lang/nl/fieldtypes.php:27]: Cool. Is it possible to edit a PR? I’d like to fix those asap.

Comment (edalzell) [reply resources/lang/nl/fieldtypes.php:27]: Yes Rob, push again to the same branch

Comment (studio1902) [reply resources/lang/nl/fieldtypes.php:27]: Good to know, push to my own forked branch, will try in a minute.

**Comment by studio1902** [comment]
Thanks @ebeauchamps. Fixed it. I’ve encountered some error’s with the CLI Translator tool at the end for one of the reviews. I think it was for fieldtypes.php. That might explain why those strings were missing.

File: resources/lang/nl/fieldtypes.php [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+    ],
+    'checkboxes' => [
+        'config' => [
+            'inline' => 'Toon de checkboxes in een rij.',
+            'options' => 'Stel de array keys en hun optionele labels in',
+        ],
+    ],
+    'code' => [
+        'config' => [
+            'indent_size' => 'Stel in hoevel indentatie je wilt (in spaties).',
```
Comment (plompd) [inline resources/lang/nl/fieldtypes.php:43]: TYPO: hoevel => hoeveel

File: resources/lang/nl/markdown.php [hunk 5]
```diff
@@ -0,0 +1,57 @@
+<?php
+
+return [
+
+    'cheatsheet' => '
+<p>Markdown is een text-naar-HTML markup syntax for webredacteuren. Met Markdown kun je leesbare en eenvoudige platte tekst te schrijven dat geconverteerd wordt naar semantische en valide HTML.</p>
```
Comment (plompd) [inline resources/lang/nl/markdown.php:6]: Voorstel:
Markdown is een text-naar-HTML markup syntax for webredacteuren. Met Markdown kun je leesbare en eenvoudige platte tekst schrijven welke geconverteerd wordt naar semantische, valide HTML.

File: resources/lang/nl/messages.php [hunk 6]
```diff
@@ -0,0 +1,149 @@
+<?php
+
+return [
+    'activate_account_notification_body' => 'Je ontvangt deze e-mail, omdat we een verzoek ontvangen hebben om je wachtwoord te resetten.',
+    'activate_account_notification_subject' => 'Activeer je account',
+    'asset_container_blueprint_instructions' => 'Blueprints definiëren aanvullende aangepaste velden die te gebruiken zijn als je assets bewerkt.',
+    'asset_container_create_folder_instructions' => 'Indien ingeschakeld, is kunnen gebruikers mappen aanmaken in deze container.',
```
Comment (plompd) [inline resources/lang/nl/messages.php:7]: Voorstel:
Indien ingeschakeld kunnen gebruikers mappen aanmaken in deze container.

File: resources/lang/nl/validation.php [hunk 7]
```diff
@@ -0,0 +1,113 @@
...
+    'max' => [
+        'numeric' => 'Mag niet groter zijn dan :max.',
+        'file' => 'Mag niet groter zijn dan :max kilobytes.',
+        'string' => 'Mag niet langer zijn dan :max karakters.',
+        'array' => 'Mag niet meer dan :items bevatten.',
+    ],
+    'mimes' => 'Moet een bestand zijn van het type: :values.',
+    'mimetypes' => 'Moet een bestand zijn van het type: values.',
+    'min' => [
+        'numeric' => 'Moet minimal :min zijn.',
```
Comment (plompd) [inline resources/lang/nl/validation.php:74]: TYPO: minimal => minimaal

**Comment by ebeauchamps** [comment]
@studio1902  An issue regarding my above comments was first reported in #1640 

Missing keys were also reported in #1697  but you should check for the great work by @Konafets in #1904 
Once his PR is merged, we should be able to get more keys to translate.

By the way, great work of yours here!

**Pull request merged**